src/test/java/com/appraisal/TestData.java

```diff
@@ -0,0 +1,36 @@
+package com.appraisal;
+
+import com.appraisal.entities.Employee;
+import com.appraisal.entities.Manager;
+import com.appraisal.modules.employee.apimodels.request.AddEmployeeModel;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class TestData {
+    public static AddEmployeeModel generateEmployeeModelRequest(){
+        LocalDate now = LocalDate.now();
+        return AddEmployeeModel.builder().firstName("Test").lastName("Test").emailAddress("test@test.com").dateEmployed(now).build();
+    }
+
+    public static Employee generateEmployee(){
+        LocalDate now = LocalDate.now();
+        LocalDateTime localDateTime = now.atStartOfDay();
+        Employee employee = Employee.builder().firstName("Test").lastName("Test").emailAddress("test@test.com").dateEmployed(localDateTime).build();
+        employee.setId(1L);
+        return employee;
+    }
+
+    public static AddEmployeeModel generateEmployeeModelRequestWithManager(){
+        LocalDate now = LocalDate.now();
+        return AddEmployeeModel.builder().firstName("Test").lastName("Test").emailAddress("test@test.com").dateEmployed(now).managerId(1L).build();
```

Comment:
wrap into new lines:
```
AddEmployeeModel.builder()
.firstName("Test")
.lastName("Test")
.emailAddress("test@test.com")
.dateEmployed(now)
.managerId(1L)
.build();
```

This is more readable. Do the same for all your builders